Documentation/Testing/ProjectTesting.rst

```diff
@@ -94,6 +94,11 @@ is added to the other containers, running in the same docker network. More infor
 setups like these can be found in the `ddev documentation
 <https://ddev.readthedocs.io/en/stable/users/extend/custom-compose-files/>`_.
 
+To execute acceptance tests in this installation you've to activate this file, usually it's now appended
```

Comment:
```suggestion
To execute acceptance tests in this installation you have to activate this file, usually it is now appended
```
we try to avoid appreviations as they make texts harder to read